README.md

```diff
@@ -1 +1,69 @@
-# Capstone-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone-Project
+
+> This project is about creating a website for a company that provides the best tailoring services.
+
+
+## Built With
+
+- HTML, CSS and JS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://wassimchakib.github.io/Capstone-Project/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example:
+- Open your terminal
+- Choose the directory that you want to save your project
+- Clone the repository : 
+ • Using SSH : `git@github.com:wassimchakib/Capstone-Project.git` 
+ • Using HTTPS : `https://github.com/wassimchakib/Capstone-Project.git`
+
+### Prerequisites
+An operating system like Windows, MAC or Linux. 
+500MB of free storage available
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Wassim Chakib**
+
+- GitHub: [@wassimchakib](https://github.com/wassimchakib)
+- Twitter: [@wassim_chakib](https://twitter.com/wassim_chakib)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/wassimchakib/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Learning to code is learning to create and innovate
```

Comment:
- [ ]  Kyndly add to the "Acknowledgements" section the appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👌👌